validation/raw_values.rst

```diff
@@ -62,9 +79,25 @@ Validation of arrays is possible using the ``Collection`` constraint::
         'gender' => new Assert\Choice(array(3, 4)),
         'file' => new Assert\File(),
         'password' => new Assert\Length(array('min' => 60)),
+        'tags' => new Assert\Required(array(
+            new Assert\Type('array'),
+            new Assert\Count(array('min' => 1)),
+            new Assert\All(array(
+                new Assert\Collection(array(
+                    'slug' => new Assert\Required(array(
+                        new Assert\NotBlank(),
+                        new Assert\Type(array('type' => 'string'))
+                    )),
+                    'label' => new Assert\Required(array(
+                        new Assert\NotBlank(),
+                    )),
+                )),
+            )),
+            new CustomUniqueTagValidator(array('groups' => 'custom'))
```

Comment:
I would move this one in the `All` constraint... I expect each tag to be unique, thus validated by this custom unique validator.